app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author_id, class_name: 'User', foreign_key: :author_id_id
```

Comment:
- [ ] Kindly change this code from `author_id` to `author` as an entity. A post should belong to an author (the post creator, not the ID).